app/models/load_balancer.rb

```diff
@@ -20,7 +20,7 @@ class LoadBalancer < ApplicationRecord
   has_many :network_ports, :as => :device
   has_many :cloud_subnet_network_ports, :through => :network_ports
   has_many :cloud_subnets, :through => :cloud_subnet_network_ports
-  has_many :floating_ips, :through => :network_ports
+  has_many :floating_ips, :through => :network_ports, :source => :floating_ip
```

Comment:
This change is weird, but I think the problem actually stems from this: https://github.com/ManageIQ/manageiq/blob/3d46ba5475173ce7d088155fc16a88965cf54cac/app/models/network_port.rb#L13-L15

Reply:
I actually think the change you've made could break the code if the expectation is that the NetworkPort#floating_ips is supposed to be used over NetworkPort#floating_ip.

Need @Ladas for some clarification here.

Reply:
Ah, cool, thanks!  Good 👁 .   I even looked at network port and didn't see that.  That makes perfect sense.  Either way, I agree, we'll need @Ladas to tell us which one is correct.

Reply:
hm interesting, I am pretty sure the through was using the ```:floating_ips``` before, so should we keep it the same?

``` has_many :floating_ips, :through => :network_ports, :source => :floating_ips``` ?

Then I am not sure if we can remove the comment, e.g. in OpenStack and AWS, it's not possible to have multiple floating_ips for interface, but i guess it can be possible somewhere?

Reply:
I'll change it to floating_ips.  Does it make sense to remove the `has_one :floating_ip` in the NetworkPort model?  I believe we've handled this before in other models by using a has_many that returns an array of 1 item.

Reply:
hm, it might be safer to remove it in another PR, since it might be use somewhere (and not sure if UI has specs)

Reply:
yeah, sorry, that's what I meant.  I think this PR is ready to go as is.